src/NotificationEventMailing.php

```diff
@@ -496,7 +496,7 @@ private static function attachDocuments(GLPIMailer $mmail, array $documents_ids)
             if (Document::isImage($path)) {
                 $path = Document::getImage(
                     $path,
-                    'mail'
+                    'mailattachment'
```

Comment:
Is the `mail` context still used ?

Reply:
Yes in ```NotificationEventMailing``` (2 times)

Reply:
If we decide to not compress images anymore in notifications, we should do it for all images. In this case, the `Document::getImage()` method will not be used anymore and could be deprecated.

Reply:
My only suggestion is that images in attachments should not be compressed.

I'm fine with resizing the email to make it easier to read